src/fides/api/ops/service/saas_request/override_implementations/authentication_strategy_adobe_campaign.py

```diff
@@ -55,7 +55,7 @@ def add_authentication(
         access_token: Optional[str] = secrets.get("access_token")
         expires_at: Optional[int] = secrets.get("expires_at")
 
-        if not access_token or self._close_to_expiration(expires_at, connection_config):
+        if not access_token or self.close_to_expiration(expires_at, connection_config):
```

Comment:
Underscores at the start of variable names aren't allowed by RestrictedPython. The custom functions under `override_implementations` aren't loaded with RestrictedPython right now but I just want to make sure we can at some point if needed